src/components/Chart/ChartTheme.tsx

```diff
@@ -1,41 +0,0 @@
-import React, { ReactNode } from "react";
-
-import {
-  Provider as FluentUIThemeProvider,
-  mergeThemes,
-  ComponentVariablesObject,
-} from "@fluentui/react-northstar";
-
-import { ThemePrepared } from "@fluentui/styles";
-
-export interface IChartThemeProps {
-  globalTheme: ThemePrepared;
-  children: ReactNode;
-}
-
-const getLocalTheme = () => {
-  return {
-    componentStyles: {
-      Box: {
-        root: ({ theme }: ComponentVariablesObject) => ({
-          "--charts-axes-labels-fg":
```

Comment:
AFAIK this is a meaningful custom property and should not be removed.

Reply:
understand. I have restored the files.